cellpack/autopack/Compartment.py

```diff
@@ -1096,44 +1074,6 @@ def createSurfacePoints(self, maxl=20):
         self.ogsurfacePoints = points
         self.ogsurfacePointsNormals = normals
 
-    def one_rapid_ray(self, pt1, pt2, diag):
-        # return number of triangle /triangle contact
-        helper = autopack.helper
-        rm = self.get_rapid_model()
-        v1 = numpy.array(pt1)
-        direction = helper.unit_vector(pt2 - pt1) * diag
-        v2 = v1 + direction
-        if sum(v1) == 0.0:
-            v3 = v2 + numpy.array([0.0, 1.0, 0.0])
-        else:
-            vcross = numpy.cross(v1, v2)  # v1 and v2 are parrallele
-            if sum(vcross) == 0:
-                v3 = v2 + numpy.array([0.0, 1.0, 0.0])
-            else:
-                v3 = v2 + helper.unit_vector(numpy.cross(v1, v2))
-        f = [0, 1, 2]
-        ray_model = RAPIDlib.RAPID_model()
-        ray_model.addTriangles(
-            numpy.array([v1, v2, v3], "f"),
-            numpy.array(
-                [f],
-                "i",
-            ),
-        )
-        RAPIDlib.RAPID_Collide_scaled(
-            numpy.identity(3),
-            numpy.array([0.0, 0.0, 0.0], "f"),
-            1.0,
-            rm,
-            numpy.identity(3),
-            numpy.array([0.0, 0.0, 0.0], "f"),
-            1.0,
-            ray_model,
-            RAPIDlib.cvar.RAPID_ALL_CONTACTS,
-        )
-        # could display it ?
-        return RAPIDlib.cvar.RAPID_num_contacts
-
     def checkPointInside_rapid(self, point, diag, ray=1):
```

Comment:
shouldnt this function be removed ?

Reply:
It is referenced a few times:
1) cellpack/autopack/BaseGrid.py Line 408 `inside = self.checkPointInside_rapid(`
2) cellpack/autopack/BaseGrid.py Line 939 `r = self.checkPointInside_rapid(coord, diag, ray=ray)`
3) cellpack/autopack/Compartment.py Line 1856 `inside = self.checkPointInside_rapid(coord, diag, ray=3)`
4) cellpack/autopack/Compartment.py Line 1875 `inside = self.checkPointInside_rapid(coord, diag, ray=3)`
5) cellpack/autopack/Compartment.py Line 3832 `r = self.checkPointInside_rapid(coord, diag, ray=ray)`
6) cellpack/autopack/ingredient/Ingredient.py Line 2191 `inside = organelle.checkPointInside_rapid(point, self.env.grid.diag, ray=3)`
7) cellpack/autopack/ingredient/Ingredient.py Line 2202 `inside = o.checkPointInside_rapid(point, self.env.grid.diag, ray=3)`

These should probably also be removed if this function is going to be removed.

Reply:
yes, I removed this and replaced calls to it with `checkPointInside`